lte/gateway/python/integ_tests/s1aptests/util/traffic_util.py

```diff
@@ -125,9 +126,13 @@ def exec_command(self, command):
 
     def update_dl_route(self, ue_ip_block):
         """ Update downlink route in TRF server """
+        #ret_code = self.exec_command(
+        #    "sudo ip route flush via 192.168.129.1 && sudo ip route "
+        #    "replace " + ue_ip_block + " via 192.168.129.1 dev eth2",
+        #)
         ret_code = self.exec_command(
-            "sudo ip route flush via 192.168.129.1 && sudo ip route "
-            "replace " + ue_ip_block + " via 192.168.129.1 dev eth2",
+            "sudo ip -6 route flush via 3001::1 && sudo ip -6 route "
```

Comment:
I think we are mixing IPv4 and IPv6 here.. The ue_ip_block here is **"192.168.128.0/24"**

Reply:
Addressed